### DISCUSSION TITLE
Exclude items in system backup bundle ?

### DISCUSSION THREAD

**Discussion by nlindblo** [longhorn/longhorn#7918] [General]
Is it possible to control what's included in the system backup bundle ? I would like to exclude recurring-jobs from being restored from a system backup and recreate these manually after restore. The background to this is that I dont want them to run while the restore of volumes are still in progress and not reinstate them until the system is fully restored

**Comment by DamiaSan** [comment]
If you recreate your volumes manually, why do you need backup/restore?

**Reply by nlindblo** [reply]
Perhaps I'm missing something here, but I don't recreate the volumes manually and I let the system restore do that. However, I have noticed that the recurring jobs gets added before the restores completed and I get hourly backups running (depending on timing) while the volume restores are still ongoing. I would prefer if the volume restores are up with my Kubernetes pods and possible re-balancing of volumes are complete before the recurring jobs are put back on. Note that this is my home lab environment, so i'm not too concerned about data loss etc.
